### PR TITLE
Added IOSXE parser 'show switch stack-ports summary'

### PR DESCRIPTION
**Pull request by smitsialis**
## Description
Added IOSXE parser for:
* show switch stack-ports summary

## Motivation and Context
New Parser

## Impact (If any)
N/A

## Screenshots:
image

## Checklist:

- [x ] I have updated the changelog.
- [ ] I have updated the documentation (If applicable).
- [x ] I have added tests to cover my changes (If applicable).
- [x ] All new and existing tests passed.
- [ ] All new code passed compilation.

### CODE REVIEW

File: src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+expected_output = {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py:1]: Please add empty test as well. like below one. just folder with empty file.
https://github.com/CiscoTestAutomation/genieparser/tree/master/src/genie/libs/parser/iosxe/tests/ShowAccessLists/cli/empty

**Review by tahigash — CHANGES_REQUESTED** [review]
reviewed schema and left comments

**Comment by xiaoxinz-cisco** [comment]
Hi @smitsialis , a friendly reminder of this PR, thanks.

**Comment by smitsialis** [comment]
@xiaoxinz-cisco  Hi,  Sorry for the delay.  Requested changes made; all tests passed.  I'm new to this and wonder how to I update the PR?

File: src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+         "link_ok":"Yes",
+         "link_active":"Yes",
+         "sync_ok":"Yes",
+         "link_changes_count":"1",
+         "in_loopback":"No"
+      },
+      "5/2":{
+         "stackport_id":"5/2",
+         "port_status":"OK",
+         "neighbor":"1",
```
[the remark this answers is not in the document]
Comment (tahigash) [reply src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py]: confirmed the update

File: src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+         "cable_length":"100cm",
+         "link_ok":"Yes",
+         "link_active":"Yes",
+         "sync_ok":"Yes",
+         "link_changes_count":"1",
```
[the remark this answers is not in the document]
Comment (tahigash) [reply src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py]: confirmed the change

File: src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py (hunk 1 above)
Comment (tahigash) [reply src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py:1]: confirmed the change

**Review by tahigash — APPROVED** [review]
@smitsialis thanks for you updates. schema looks good now and approved. and I fixed conflicts for you.

@xiaoxinz-cisco please review and if ok, please approve and merge.

**Review by xiaoxinz-cisco — APPROVED** [review]

**Pull request merged**